docs/sphinx/source/whatsnew/v0.9.3.rst

```diff
@@ -12,6 +12,9 @@ Enhancements
 * Low resolution altitude lookup map
   :py:func:`~pvlib.location.lookup_altitude`
   (:issue:`1516`, :pull:`1518`)
+* New module to calculate spectral mismatch
+  :py:func:`~pvlib.spectrum.mismatch.calc_spectral_mismatch_field`
+  (:issue:`1523`, :pull:`1524`)
```

Comment:
Will do.

Reply:
Regarding your other comment on offering `sr_ref`: it's a reasonable suggestion.  Perhaps for a more general companion function combined with other advanced options?  Comparing two devices can also be easily and intuitively done by running this function twice and taking the quotient.

Reply:
My thought was that, if more general functionality will be added eventually and one general function is better than two partially overlapping functions, we might want to avoid locking ourselves into the `calc_spectral_mismatch_field` name.  But clearly others are more familiar than I am with the typical workflow here, so I defer to your judgement on what makes the most sense.

Reply:
The name could be changed, if it that helps.  There is a quite range of possibilities for making a more general function.  I wouldn't claim that this makes the most sense but it gets us one step further in the general direction.